auth/src/main/kotlin/com/hedvig/android/auth/AuthRepository.kt

```diff
@@ -32,23 +33,26 @@ sealed interface AuthAttemptResult {
 
   data class BankIdProperties(
     val id: String,
-    val statusUrl: String,
+    val statusUrl: StatusUrl,
     val autoStartToken: String,
   ) : AuthAttemptResult
 
   data class ZignSecProperties(
     val id: String,
-    val statusUrl: String,
+    val statusUrl: StatusUrl,
     val redirectUrl: String,
   ) : AuthAttemptResult
 
   data class OtpProperties(
     val id: String,
-    val statusUrl: String,
+    val statusUrl: StatusUrl,
     val validationUrl: String,
   ) : AuthAttemptResult
 }
 
+@JvmInline
+value class StatusUrl(val url: String)
```

Comment:
Love this way of avoiding passing around error-prone primitives on APIs like this 🤩